<?xml version='1.1' encoding='UTF-8'?>
<Tag plugin="workflow-support@839.v35e2736cfd5c">
  <node class="cps.n.StepEndNode" plugin="workflow-cps@3705.va_6a_c2775a_c17">
    <parentIds>
      <string>152</string>
    </parentIds>
    <id>153</id>
    <startId>150</startId>
  </node>
  <actions>
    <wf.a.TimingAction plugin="workflow-api@1232.v1679fa_2f0f76">
      <startTime>1690492023564</startTime>
    </wf.a.TimingAction>
    <wf.a.ErrorAction plugin="workflow-api@1232.v1679fa_2f0f76">
      <error class="hudson.AbortException">
        <detailMessage>script returned exit code 1</detailMessage>
        <stackTrace>
          <trace>org.jenkinsci.plugins.workflow.steps.durable_task.DurableTaskStep$Execution.handleExit(DurableTaskStep.java:667)</trace>
          <trace>org.jenkinsci.plugins.workflow.steps.durable_task.DurableTaskStep$Execution.check(DurableTaskStep.java:613)</trace>
          <trace>org.jenkinsci.plugins.workflow.steps.durable_task.DurableTaskStep$Execution.run(DurableTaskStep.java:557)</trace>
          <trace>java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)</trace>
          <trace>java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)</trace>
          <trace>java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)</trace>
          <trace>java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)</trace>
          <trace>java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)</trace>
          <trace>java.base/java.lang.Thread.run(Thread.java:829)</trace>
        </stackTrace>
        <suppressedExceptions>
          <wf.a.ErrorAction_-ErrorId>
            <stackTrace/>
            <suppressedExceptions class="empty-list"/>
            <uuid>ed685107-3927-4667-af64-58c631af77be</uuid>
          </wf.a.ErrorAction_-ErrorId>
        </suppressedExceptions>
      </error>
    </wf.a.ErrorAction>
  </actions>
</Tag>